client/src/workspace/layouts/AppsViewLayout.tsx

```diff
@@ -18,12 +18,22 @@ export const AppsViewLayout: React.FC = () => {
   const { data } = useAppsListing();
 
   const portalApp = findAppById(data, app.definition.id);
-
   const icon = portalApp?.icon || app.definition.notes.icon || 'Generic-App';
-  const userGuideLink =
-    portalApp?.userGuideLink || app.definition.notes.helpUrl;
 
   const htmlApp = data?.htmlDefinitions[appId];
+  const rawHtml = (htmlApp?.html as string) || '';
+
+  const userGuideLink =
+    htmlApp?.userGuideLink ||
+    portalApp?.userGuideLink ||
+    app.definition.notes.helpUrl;
+
+  // Remove any btn-styled links (User Guide buttons) from HTML blob
+  const filteredHtml = rawHtml.replace(
+    /<a[^>]*class=["'][^"']*btn[^"']*["'][^>]*>.*?<\/a>/gi,
+    ''
+  );
```

Comment:
Rather than this we should just go through the html of these apps and remove these buttons